pkgs/development/tools/jsonnet-bundler/default.nix

```
@@ -0,0 +1,28 @@
+{ buildGoModule, fetchFromGitHub, lib }:
+
+buildGoModule rec {
+  pname = "jsonnet-bundler";
+  version = "0.4.0";
+
+  src = fetchFromGitHub {
+    owner = "jsonnet-bundler";
+    repo = "jsonnet-bundler";
+    rev = "v${version}";
+    sha256 = "0pk6nf8r0wy7lnsnzyjd3vgq4b2kb3zl0xxn01ahpaqgmwpzajlk";
+  };
+
+  vendorSha256 = "1h1d32zi1pfb7spprsacxg8ygrigswqqngrc8gsdzn1xbldphl5g";
+
+  doCheck = true;
+
+  subPackages = [ "cmd/jb" ];
+
+  buildFlagsArray = [ "-ldflags=-s -w -X main.Version=${version}" ];
+
+  meta = with lib; {
+    description = "A jsonnet package manager.";
```

```suggestion
    description = "A jsonnet package manager";
```
`meta.description` should not have a period at the end, see [contributing guide](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).